### ISSUE TITLE
`Squad` method refers to `mp_spent`, `sp_spent`, which are not in `Squad` or any of its base classes

### ISSUE THREAD

**Issue by AmkG** [jwvhewitt/gearhead-caramel#63]
`mp_spent` and `sp_spent` are initialized only in `class Being`:

https://github.com/jwvhewitt/gearhead-caramel/blob/32a0b43ec33f33eeafd98f782194dc0e5cad76f8/gears/base.py#L3179-L3180

Only `Character` seems to derive from `Being`.  `class Squad` derives from `BaseGear`, not `Being`.

These methods in `Squad` refer to `mp_spent` and `sp_spent`:

https://github.com/jwvhewitt/gearhead-caramel/blob/32a0b43ec33f33eeafd98f782194dc0e5cad76f8/gears/base.py#L3631-L3641

It *looks* to me that `Squad` is not used currently, so maybe it is a "TODO" class?

**Comment by jwvhewitt** [comment]
You are correct, Squad is not currently being used. I was planning to use it for the player's lance on the world map and possibly also for fighting teams of human scale combatants in mecha combat (power armor like the Strongarm and Badger, or biomonsters).
